### PR TITLE
add map command

### PR DESCRIPTION
**Pull request by harin**

### CODE REVIEW

File: cmd.js [hunk 1]
```diff
@@ -110,5 +112,38 @@ module.exports = Command = {
...
+    if (activeDevice) {
+      const currentFloor = activeDevice.floor
+      return currentFloor
+    }
+    return null
+  },
+  getOfficeMap: async function () {
+    const names = await User.distinct('name')
+    const users = []
+    await Promise.all(
```
Comment (hampallop) [inline cmd.js]: Since `map` already return an array, can we assign `users` like....?
```js
const users = await Promise.all(...)
```

Comment (harin) [reply cmd.js]: 👍

Comment (hampallop) [reply cmd.js]: Okay, no need to change I forgot the data in `users` should exclude `floor == null`.

**Comment by harin** [comment]
Example output
```
{
	"attachments": [
		{
			"title": "Absent",
			"text": ":armfusrodah: :awe:  :question:  "
		},
		{
			"title": "4th Floor",
			"text": ":airblackhat: :birdboom: :boat: :bookthug2: :goodthug: :groupfatboy: :hamsome:  :netwander:  :nuttrollface: Σ  :yingwithglasses:"
		},
		{
			"title": "3rd Floor",
			"text": ":benzsmileyface: :booknwat: :heart: :mamieo: :g:  :apple: :jibmove: :johnny: :nongkenredhair: :khame2: :wat: :pong:"
		},
		{
			"title": "2nd Floor",
			"text": ":ahd: :iphone: :video_camera:"
		},
		{
			"title": "1st Floor",
			"text": ":hypnotoad2: :google: :hunt: :saimaikid: :trawuthandsome:"
		}
	]
}
```

**Review by hampallop — APPROVED** [review]
I've tested. Great job! and Thank you for your contribution!! 👍

**Pull request merged**